streaming/src/main/scala/org/apache/spark/streaming/util/FileBasedWriteAheadLogSegment.scala

```diff
@@ -17,4 +17,5 @@
 package org.apache.spark.streaming.util
 
 /** Class for representing a segment of data in a write ahead log file */
-private[streaming] case class WriteAheadLogFileSegment (path: String, offset: Long, length: Int)
+private[streaming] case class FileBasedWriteAheadLogSegment(path: String, offset: Long, length: Int)
```

Comment:
Is it better to rename to `FileBasedWriteAheadLogRecordHandle`, here is `Segment`, but the base class is `RecordHandle`, is is better to keep consistent?